### PR TITLE
Improve auto

### PR DESCRIPTION
**Pull request by hannasoderstrom1**
Writes if GMS-BT or GMS-AL in email. Writes if prio in email.
End mail includes which samples have finished correctly and which have not.

### CODE REVIEW

File: tools/email.py [hunk 1]
```diff
@@ -54,8 +54,24 @@ def end_email(run_name, samples):
...
+                      f'{new_line}{new_line.join(ok_samples)}',
+                      '',
+                      'The following samples have not finished correctly:',
+                      '',
+                      f'{new_line}{new_line.join(not_ok_samples)}',
+                      '',
+                      'Errors concerning these samples will be investigated.',
+                      '',
+                      'Best regards,',
+                      'CGG Cancer'])
```
Comment (alvaralmstedt) [inline tools/email.py:73]: I'd use triple quotes text block:
    """
    text here
    """

File: tools/email.py [hunk 2]
```diff
@@ -13,8 +13,8 @@ def send_email(subject, body):
 
     msg['Subject'] = subject
     msg['From'] = "cgg-cancer@gu.se" # TODO Get from config
-    msg['To'] = "cgg-cancer@gu.se" # TODO Get from config and have different recipients for errors and success
-    msg['Cc'] = "hanna.soderstrom@gu.se" # TODO Get from config
+    msg['To'] = "gms_btb@gu.se" # TODO Get from config and have different recipients for errors and success
```
Comment (alvaralmstedt) [inline tools/email.py:16]: Kommer vi inte få 2 mail?

File: wgs_somatic-run-wrapper.py [hunk 3]
```diff
@@ -217,15 +229,28 @@ def wrapper(instrument):
             u.join()
             logger.info(f'Thread {u} is finished')
 
+        ok_samples = []
+        not_ok_samples = []
```
Comment (alvaralmstedt) [inline wgs_somatic-run-wrapper.py:233]: maybe rename to bad_samples?

Comment (alvaralmstedt) [reply wgs_somatic-run-wrapper.py:233]: or something

**Review by alvaralmstedt — COMMENTED** [review]
Is it tested?

File: tools/email.py (hunk 1 above)
Comment (hannasoderstrom1) [reply tools/email.py:73]: how do I handle the f strings if I make a text block instead?

File: tools/email.py (hunk 2 above)
Comment (hannasoderstrom1) [reply tools/email.py:16]: verkar inte så. det är så här nu också (fast inte committat) och vi får ändå bara ett mail

File: wgs_somatic-run-wrapper.py (hunk 3 above)
Comment (hannasoderstrom1) [reply wgs_somatic-run-wrapper.py:233]: sure. solved in commit @ 176f570

File: tools/email.py (hunk 1 above)
Comment (hannasoderstrom1) [reply tools/email.py:73]: solved in commit @ 785922c

**Pull request merged**